cinema/serializers.py

```diff
@@ -33,7 +39,9 @@ class Meta:
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = ("id", "title", "description", "duration",
+                  "genres", "actors", "image")
```

Comment:
the best practice is to move each field on a separate line like you do with imports when it's longer than 79 characters ;)